mediaire_toolbox/transaction_db/index.py

```diff
@@ -30,3 +33,47 @@ def index_study_date(transaction):
     except Exception:
         study_date = ''
     transaction.study_date = study_date
+
+
+def index_version(transaction):
```

Comment:
I was confused at the beginning if these methods are getters or setters, I would maybe write them as "set_index_version" and write a doc header that says what the method does. And I think it would be more understandable that this method belongs to the Transaction object rather than separating it (even though the file will be longer).

Reply:
will change the function names and add the docs.
However i dont really think they should belong to the Transaction object since they are essentially migration scripts:
they are called only once, and then they served their purpose.
We have a migration.py file, perhaps we can move these functions to this file